jupyter-notebooks/orders_basemaps/SDK_order_basemaps.ipynb

```diff
@@ -0,0 +1,471 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "mosaic_id = basemaps['mosaics'][0]['id']\n",
+    "mosaic_box = basemaps['mosaics'][0]['bbox']\n",
+    "mosaic_box_str = ','.join(map(str, mosaic_box))\n",
+    "\n",
+    "quad_params = {\n",
+    "    'mosaic_id': mosaic_id,\n",
+    "    'bbox': mosaic_box_str,\n",
+    "}\n",
+    "\n",
+    "quads_url = \"{}/{}/quads\".format(BASEMAP_API_URL, mosaic_id)\n",
+    "quad_service_response = requests.get(\n",
+    "    url=quads_url, params=quad_params, auth=auth, headers=headers)\n",
+    "quads = quad_service_response.json()\n",
+    "items = quads['items']\n",
+    "\n",
+    "quad_ids = [i['id'] for i in items]"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Create an order packet\n",
+    "---\n",
+    "Package up the details of your order in a [JSON object](https://developers.planet.com/apis/orders/basemaps/#example-order-query-json-block) and make a POST request, passing in the Orders URL, your JSON, your API key, and the content-type header. We are [delivering](https://developers.planet.com/apis/orders/delivery/) this order to a Google Cloud Storage bucket. In this example, we are using the tools [bandmath](https://developers.planet.com/apis/orders/tools/#band-math) and [reproject](https://developers.planet.com/apis/orders/tools/#reproject)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Reproject\n",
+    "\n",
+    "Reprojecting a basemap is necessary when the original map projection does not accurately represent the Earth's surface in a specific region or area. This can happen when the imagery for the basemap comes from a place far from the equator. In the example shown in this notebook, there is not a significant change in circumference of the earth from the equator, but it is always best to check using the code below. \n",
+    "\n",
+    "First, find the latitude of the area of interest. Then, find the radius of the Earth in kilometers. We recommend using a website like [this](https://rechneronline.de/earth-radius/) to find that number. Simply change the latitude and earth radius parameters."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def calculate_resolution_at_latitude(latitude, earth_radius):\n",
+    "    '''Calculate the resolution of a basemap at a given latitude\n",
+    "\n",
+    "    Parameters: latitude (float): latitude at location of interest\n",
+    "                native_resolution (float): native resolution of basemap\n",
+    "    Return (float): resolution of basemap at given latitude\n",
+    "    '''\n",
+    "    earth_radius = 6370.1 # km \n",
```

Comment:
So, you've got `earth_radius` in the input and hard-coded here. I'd recommend just leaving it here and removing it from the inputs.